lua/lvim/core/autocmds.lua

```diff
@@ -180,16 +180,23 @@ function M.toggle_format_on_save()
 end
 
 function M.enable_reload_config_on_save()
-  local user_config_file = require("lvim.config"):get_user_config_path()
+  local pattern = get_config_dir() .. "/*.lua"
+
+  -- add symlinked path to the pattern
+  local linked_config_lua = vim.fn.resolve(join_paths(get_config_dir(), "config.lua"))
+  if linked_config_lua ~= join_paths(get_config_dir(), "config.lua") then
+    local linked_config_dir = linked_config_lua:sub(1, #linked_config_lua - #"config.lua")
+    pattern = pattern .. "," .. linked_config_dir .. "*.lua"
+  end
 
   if vim.loop.os_uname().version:match "Windows" then
```

Comment:
```suggestion
```

since this is a requirement either way, there's no need to check if it's on windows or not :)